tdms/src/iterator.cpp

```diff
@@ -4556,7 +4520,7 @@ void execute_simulation(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs
   *mxGetPr((mxArray *) plhs[25]) = maxfield;
 
   if (params.run_mode == RunMode::complete && params.exphasorsvolume) {
-    setGridLabels(input_grid_labels, output_grid_labels, E.il, E.iu, E.jl, E.ju, E.kl, E.ku);
+    output_grid_labels.initialise_from(input_grid_labels, E.il, E.iu, E.jl, E.ju, E.kl, E.ku);

```

Comment:
I now maybe understand initialise != construct, but I leave my previous question just in case.